packit_service/worker/helpers/testing_farm.py

```diff
@@ -215,6 +215,7 @@ def custom_fmf(self) -> bool:
     def fmf_url(self) -> str:
         return (
             self.job_config.fmf_url
+            or (self.job_config.use_target_repo_for_fmf_url and self.project.get_web_url())
             or (self.pull_request_object and self.pull_request_object.source_project.get_web_url())
             or self.project.get_web_url()
```

Comment:
I suppose it's subjective, but this seems more readable to me:

```suggestion
            self.job_config.fmf_url
            or (
                self.pull_request_object
                and not self.job_config.use_target_repo_for_fmf_url
                and self.pull_request_object.source_project.get_web_url()
            )
            or self.project.get_web_url()
```